src/main/java/seedu/task/Task.java

```diff
@@ -0,0 +1,58 @@
+package task;
+
+public class Task {
+
+    private final String description;
+    private boolean completed;
```

Comment:
```suggestion
    private boolean isCompleted;
```
Would isCompleted be a more intuitive vairable name?